<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetDISetting</class>
 <widget class="QWidget" name="WidgetDISetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Item</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>DI</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>id</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="customSpinBox" name="spinBox_id">
        <property name="toolTip">
         <string>范围[0,15]</string>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_prompt">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Setting</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_func">
          <property name="toolTip">
           <string>功能</string>
          </property>
          <property name="text">
           <string>func</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="customComboBox" name="comboBox_func">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_type">
          <property name="toolTip">
           <string>设备类型</string>
          </property>
          <property name="text">
           <string>type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="customComboBox" name="comboBox_type">
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>range[°]</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_range">
            <property name="toolTip">
             <string>范围[0,360]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>maxDist[m]</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_maxDist">
            <property name="toolTip">
             <string>范围[minDist,100]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>minDist[m]</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_minDist">
            <property name="toolTip">
             <string>范围[0,maxDist]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_inverse">
          <property name="toolTip">
           <string>逻辑反相</string>
          </property>
          <property name="text">
           <string>inverse</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="customCheckBox" name="checkBox_inverse"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Location</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_x">
          <property name="toolTip">
           <string>在机器人坐标系的x偏移</string>
          </property>
          <property name="text">
           <string>x[m]</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="customDoubleSpinBox" name="doubleSpinBox_x">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.200000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_y">
          <property name="toolTip">
           <string>在机器人坐标系的y偏移</string>
          </property>
          <property name="text">
           <string>y[m]</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="customDoubleSpinBox" name="doubleSpinBox_y">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_z">
          <property name="toolTip">
           <string>在机器人坐标系的z偏移</string>
          </property>
          <property name="text">
           <string>z[m]</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="customDoubleSpinBox" name="doubleSpinBox_z">
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="accelerated">
           <bool>false</bool>
          </property>
          <property name="keyboardTracking">
           <bool>true</bool>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-10.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_r">
          <property name="toolTip">
           <string>在机器人坐标系的朝向角</string>
          </property>
          <property name="text">
           <string>r[°]</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="customDoubleSpinBox" name="doubleSpinBox_r">
          <property name="toolTip">
           <string>范围[-180,180]</string>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>-180.000000000000000</double>
          </property>
          <property name="maximum">
           <double>180.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customComboBox</class>
   <extends>QComboBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
  <customwidget>
   <class>customDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
  <customwidget>
   <class>customSpinBox</class>
   <extends>QSpinBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
  <customwidget>
   <class>customCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
